### DISCUSSION TITLE
masteries

### DISCUSSION THREAD

**Discussion by kfishkin** [LukeCroteau/RaidExtractor#58] [Q&A] [answered]
hi everybody,
using the extractor and doing extractions before-and-after adding a mastery, i've figured out the codes for most of the masteries. Here's what I've got, hope you find it useful - it'd be great if folks posted other ones as they find them:
---------------------------------
Arcane Celerity:500334
Blade Disciple:500112
Blastproof:undefined
Blood Shield:undefined
Bloodthirst:500233
Bring It Down:500141
Bulwark:undefined
Charged Focus:500324
Cycle of Magic:undefined
Cycle of Revenge:undefined
Cycle of Violence:undefined
Deadly Precision:500113
Defiant:500213
Delay Death:undefined
Deterrence:undefined
Eagle Eye:500364
Elixir of Life:500361
Evil Eye:500344
Exalt in Death:500323
Fearsome Presence:undefined
Flawless Execution:undefined
Giant Slayer:500163
Grim Resolve:undefined
Harvest Despair:undefined
Healing Savior:500331
Heart of Glory:undefined
Helmsmasher:undefined
Improved Parry:500224
Iron Skin:500261
Keen Strike:500122
Kill Streak:500152
Lasting Gifts:undefined
Lay on Hands:500321
Life Drinker:500132
Lore of Steel:500343
Master Hexer:500354
Merciful Aid:undefined
Methodical:500151
Mighty Endurance:undefined
Opportunist:undefined
Oppressor:undefined
Pinpoint Accuracy:500313
Rapid Response:500332
Rejuvenation:undefined
Resurgent:undefined
Retribution:500253
Ruthless Ambush:undefined
Selfless Defender:undefined
Shadow Heal:undefined
Shield Breaker:500123
Shieldbearer:undefined
Single Out:500131
Sniper:500353
Solidarity:undefined
Spirit Haste:undefined
Steadfast:500312
Stoked to Fury:undefined
Stubbornness:undefined
Swarm Smiter:500333
Timely Intervention:undefined
Tough Skin:500212
Unshakeable:500264
Warmaster:500161
Whirlwind of Death:500133
Wisdom of Battle:undefined
Wrath of the Slain:undefined

**Comment by zerfl** [comment, marked as answer]
This is a list of all masteries and their IDs:
```
500112:Blade Disciple
500113:Deadly Precision
500121:Heart of Glory
500122:Keen Strike
500123:Shield Breaker
500124:Grim Resolve
500131:Single Out
500132:Life Drinker
500133:Whirlwind of Death
500134:Ruthless Ambush
500141:Bring It Down
500142:Wrath of the Slain
500143:Cycle of Violence
500144:Opportunist
500151:Methodical
500152:Kill Streak
500153:Blood Shield
500154:Stoked To Fury
500161:Warmaster
500162:Helmsmasher
500163:Giant Slayer
500164:Flawless Execution
500212:Tough Skin
500213:Defiant
500221:Blastproof
500222:Rejuvenation
500223:Mighty Endurance
500224:Improved Parry
500231:Shadow Heal
500232:Resurgent
500233:Bloodthirst
500234:Wisdom of Battle
500241:Solidarity
500242:Delay Death
500243:Harvest Despair
500244:Stubbornness
500251:Selfless Defender
500252:Cycle of Revenge
500253:Retribution
500254:Deterrence
500261:Iron Skin
500262:Bulwark
500263:Fearsome Presence
500264:Unshakeable
500312:Steadfast
500313:Pinpoint Accuracy
500321:Lay On Hands
500322:Shieldbearer
500323:Exalt in Death
500324:Charged Focus
500331:Healing Savior
500332:Rapid Response
500333:Swarm Smiter
500334:Arcane Celerity
500341:Merciful Aid
500342:Cycle of Magic
500343:Lore of Steel
500344:Evil Eye
500351:Lasting Gifts
500352:Spirit Haste
500353:Sniper
500354:Master Hexer
500361:Elixir of Life
500362:Timely Intervention
500363:Oppressor
500364:Eagle-Eye
```

**Reply by kfishkin** [reply]
great, thanks!

**Reply by zerfl** [reply]
Very welcome, please mark this as answered if possible.
